crates/revm/benches/burntpix/genesis_alloc.rs

```diff
@@ -0,0 +1,81 @@
+use lazy_static::lazy_static;
+use revm::primitives::{address, U256};
+use revm_precompile::{Address, Bytes, HashMap};
+use std::str::FromStr;
+
+lazy_static! {
+    pub static ref BURNTPIX_ADDRESS: Address = address!("49206861766520746f6f206d7563682074696d65");
+    pub static ref GENESIS_ALLOCS: HashMap<Address, GenesisAccount> = {
```

Comment:
Would make this just a ordinary function, as it is called just once there is no need to make it static.